### PR TITLE
added the structure of our paper

### LINKED ISSUE #2: Set-Up Structure
decide what headlines we could use

### PR DESCRIPTION
**Pull request by RaphaW**
Closes #2

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -38,6 +38,29 @@
...
+
+\section{Related work}
+
+\subsection{Generating models}
+
+\subsubsection{Image generating models}
+
+\subsubsection{Text generating models}
+
+\subsection{Generating model detectors}
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]: I would rename it to Detector models
```suggestion
\subsection{Detector models}
```

File: paper.tex [hunk 2]
```diff
@@ -38,6 +38,29 @@
...
+
+\subsubsection{Image generating model detectors}
+
+\subsubsection{Text generating model detectors}
+
+\section{Method}
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex:57]: I would name it Methodology.
```suggestion
\section{Methodology}
```

File: paper.tex [hunk 3]
```diff
@@ -38,6 +38,29 @@
...
+
+\subsection{Generating models}
+
+\subsubsection{Image generating models}
+
+\subsubsection{Text generating models}
+
+\subsection{Generating model detectors}
+
+\subsubsection{Image generating model detectors}
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
\subsubsection{Image detector models}
```

File: paper.tex [hunk 4]
```diff
@@ -38,6 +38,29 @@
...
+
+\subsubsection{Text generating model detectors}
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex]:
```suggestion
\subsubsection{Text detector models}
```

File: paper.tex [hunk 5]
```diff
@@ -38,6 +38,29 @@
...
+
+\section{Method}
+
```
Comment (EhrenbruderRoschtbraten) [inline paper.tex:58]: I think we could use a result section, where we present our results.
```suggestion

\section{Results}

```

**Review by EhrenbruderRoschtbraten — COMMENTED** [review]
Some changes I think would be nice. Some headers only renamed, one added.

File: paper.tex (hunk 2 above)
Comment (RaphaW) [reply paper.tex:57]: I would stick to method

**Review by EhrenbruderRoschtbraten — APPROVED** [review]
LGTM

**Pull request merged**